src/util.h

```diff
@@ -235,6 +243,55 @@ bool SoftSetBoolArg(const std::string& strArg, bool fValue);
 
 // Forces a arg setting, used only in testing
 void ForceSetArg(const std::string& strArg, const std::string& strValue);
+};
```

Comment:
Yes, that's exactly the point, but I have done similar things in the past and not fixed them later, something I feel bad about, for example: https://github.com/bitcoin/bitcoin/blob/master/src/validation.cpp#L1313
Please, help me make sure that it gets done this time.